src/api-reference/request/v4.endpoints.cashadvance-resources.markdown

```diff
@@ -18,9 +18,11 @@ layout: reference
 
 # <a name="request-cash-advance-resource"></a>Request Cash Advance Resources
 
-* [Get the list of existing cash advances within a Request](#get-request-cash-advance-resource)
+Note: This cash advance detail endpoint is provided within the Concur Request API for feature parity purpose only, awaiting for the Cash Advance API to be publicly externalized. It is highly recommended to only rely on the list of cash advances link available in the Request payload response, which will be updated with the latest Cash Advance API when publicly available, and not on this cash advance detail URI presented below, which will be deprecated soon after the public availability of the Cash Advance API.
```

Comment:
We cannot advertise that we are planning to release the Cash Advance API. Please remove all references to creating an additional API to correct this issue. This creates a legally binding promise to users and is not something we can advertise on the Developer Center. You may mention that this use may be deprecated in the future, but we cannot do so by promising to develop a new API.